ui/src/views/404/Footer/index.jsx

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+// This is an example of styles that are defined inline
+const style = {
+    borderTopWidth: '3px',
+    borderColor: '#0061A6',
+    borderTopStyle: 'solid',
+    paddingTop: '2px',
+    fontSize: '11px',
+    lineHeight: '14px',
+    margin: '15px 0',
+    clear: 'both',
+};
+
+const item = ({ text, ...props }) =>
+    <li
+        style={{
+            display: 'inline',
+            ':after': {
+                content: '" | "',
+            },
+        }}>
+        <a
+            {...props}
+            style={{
+                color: '#005689',
+                textDecoration: 'none',
+            }}>
```

Comment:
For me, if every element has this I would find it difficult to follow the html structure